examples/Bremsstrahlung/include/simulation_defines/param/laserConfig.param

```diff
@@ -0,0 +1,325 @@
+            constexpr float_64 TILT_X_SI = 0;
+        }
+        /** The laser pulse will be initialized PULSE_INIT times of the PULSE_LENGTH
+        *  unit: none */
+        constexpr float_64 PULSE_INIT = 20.0;
+
+        /* laser phase shift (no shift: 0.0) */
+        constexpr float_X LASER_PHASE = 0.0; /* unit: rad, periodic in 2*pi */
+
+        enum PolarisationType
+        {
+            LINEAR_X = 1u,
+            LINEAR_Z = 2u,
+            CIRCULAR = 4u
+        };
+        constexpr PolarisationType Polarisation = LINEAR_X;
+    }
+
+    namespace laserPlaneWave
+    {
+        // NOT-symetric sinus used: starts with phase=0 --> E-field=0
+        namespace SI
+        {
+            /** unit: meter */
+            constexpr float_64 WAVE_LENGTH_SI = 0.8e-6;
+
+            /** UNITCONV */
+            constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI * ::picongpu::SI::ELECTRON_MASS_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI / ::picongpu::SI::ELECTRON_CHARGE_SI;
+
+            /** unit: W / m^2 */
+            // calculate: _A0 = 8.549297e-6 * sqrt( Intensity[W/m^2] ) * wavelength[m] (linearly polarized)
+
+            /** unit: none */
+            constexpr float_64 _A0 = 1.5;
+
+            /** unit: Volt /meter */
+            constexpr float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_SI;
+
+            /** unit: Volt /meter */
+            //constexpr float_64 AMPLITUDE_SI = 1.738e13;
+
+            /** The profile of the test Lasers 0 and 2 can be stretched by a
+             *      constexprant area between the up and downramp
+             *  unit: seconds */
+            constexpr float_64 LASER_NOFOCUS_CONSTANT_SI = 13.34e-15;
+
+            /** Pulse length: sigma of std. gauss for intensity (E^2)
+             *  PULSE_LENGTH_SI = FWHM_of_Intensity   / [ 2*sqrt{ 2* ln(2) } ]
+             *                                          [    2.354820045     ]
+             *  Info:             FWHM_of_Intensity = FWHM_Illumination
+             *                      = what a experimentalist calls "pulse duration"
+             *  unit: seconds (1 sigma) */
+            constexpr float_64 PULSE_LENGTH_SI = 10.615e-15 / 4.0;
+
+        }
+
+        /** The laser pulse will be initialized half of PULSE_INIT times of the PULSE_LENGTH before and after the plateau
+         *  unit: none */
+        constexpr float_64 RAMP_INIT = 20.6146;
+
+        /* we use a sin(omega*time + laser_phase) function to set up the laser - define phase: */
+        constexpr float_X LASER_PHASE = 0.0; /* unit: rad, periodic in 2*pi */
+
+        enum PolarisationType
+        {
+            LINEAR_X = 1u,
+            LINEAR_Z = 2u,
+            CIRCULAR = 4u,
+        };
+        constexpr PolarisationType Polarisation = LINEAR_X;
+    }
+
+    namespace laserWavepacket
+    {
+    // Asymetric sinus used: starts with phase=0 at center --> E-field=0 at center
```

Comment:
changed already in main files, that whole naming made no sense at all.
out of scope for this PR (copied)